src/components/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { TOAST_DURATION_TIME } from '@/src/models/toastModel';
+import { useToast } from '@/src/stores/toastStore';
+
+import Toast from './Toast';
+
+const meta: Meta<typeof Toast> = {
+  title: 'Toast',
+  component: Toast,
+};
```

Comment:
```suggestion
const meta: Meta<typeof Toast> = {
  title: 'Components/Toast',
  component: Toast,
  tags: ['autodocs']
};
```
이렇게 추가해주시면 좋을 것 같습니다!

Reply:
@dmswl98 
https://storybook.js.org/docs/api/main-config-docs#autodocs
모든 스토리에 docs를 추가하는거라면 아예 전역에서 선언하는건 어떨까요?

Reply:
그런 설정도 있었군요...! 좋아요!! 해당 설정 부분은 [PR](https://github.com/YAPP-Github/23rd-Web-Team-2-FE/pull/20) 여기에서 추가하도록 할게요~